demos/src/demo.scss

```diff
@@ -13,11 +17,19 @@ $o-loading-is-silent: false;
 }
 
 .demo-card--light {
-	background: #f6e9d8;
+	@if (global-variable-exists(o-brand)) {
+		background: if($o-brand == 'internal', white, #fff1e5);
+	} @else {
```

Comment:
Did `@include oColorsFor(page);` not work out?

Reply:
No it didn't. `@include oColorsGetUseCase(page, background)` didn't either. I don't know if it had something to do with the way my demos kept building, but even after removing and reinstalling everything, I got nothing. If you manage to make it work locally, I'll change it.